deno.json

```
@@ -39,7 +39,7 @@
     "@std/media-types": "jsr:@std/media-types@^1.0.1",
     "@std/msgpack": "jsr:@std/msgpack@^1.0.0-rc.2",
     "@std/net": "jsr:@std/net@^0.224.4",
-    "@std/path": "jsr:@std/path@1.0.0-rc.3",
+    "@std/path": "jsr:@std/path@1.0.0",
```

Should it be this? Does this make a difference?
```suggestion
    "@std/path": "jsr:@std/path@^1.0.0",
```

---

Thanks for catching. We need that to make package deduplication possible.